packages/frontpage/app/api/notification-read/route.ts

```diff
@@ -0,0 +1,19 @@
+import { markNotificationRead } from "@/lib/data/db/notification";
+import { ensureUser } from "@/lib/data/user";
+
+export async function POST(request: Request) {
+  await ensureUser();
```

Comment:
we should probably get the user id and pass that along too and then reconfirm it's the user in the backend (which we are already doing)